src/sdk/account/toNexusAccount.ts

```diff
@@ -345,6 +356,24 @@ export const toNexusAccount = async (
     }
   }
 
+  /**
+   * @description Changes the active module for the account
+   * @param newModule - The new module to set as active
+   * @returns void
+   */
+  const setActiveValidationModule = (validationModule: BaseValidationModule): void => {
+    defaultedActiveModule = validationModule;
+  }
+
+  /**
+   * @description Gets the active validation module for the account
+   * @returns The active validation module
+   */
+  const getActiveValidationModule = (): BaseValidationModule => {
+    return defaultedActiveModule;
+  }
```

Comment:
remove?

Reply:
Unless this brakes something I would keep it, otherwise we cannot offer a way to get current active module on the SA through sdk